xinference/api/restful_api.py

```diff
@@ -2271,6 +2271,11 @@ async def create_ocr(
         self,
         model: str = Form(...),
         image: UploadFile = File(media_type="application/octet-stream"),
+        prompt: str = Form(None),
+        model_size: str = Form(None),
+        save_results: bool = Form(None),
+        save_dir: Optional[str] = Form(None),
```

Comment:
`save_dir` is a directory that exists in server, this should not be exposed to users.